dali/pipeline/basic/coords.cc

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) 2018, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "dali/pipeline/basic/coords.h"
+
+namespace dali {
+namespace basic {
```

Comment:
Wouldn't it be good idea to keep these 2 namespaces separate? Thinking about it from the "user" point of view (by "user" I regard a developer who wants to write a custom operator for himself), he doesn't really care what's inside dali  "engine". He only cares about the operator. Furthermore, the `basic` namespace is supposed to be separate library, it would be nice distinction also

Reply:
More on this: What is the basic namespace going to contain? Name seems a bit vague
I think that having a root namespace dali is a must, but I also encourage putting all the pipeline stuff in dali::pipeline and the rest in what is not dali::basic but with a different name